lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -32,6 +32,14 @@
   min-width: $nx-page-width-min;
 }
 
+.nx-page--sidebar {
+  @include container-vertical;
```

Comment:
But isn't in a horizontal container?

Reply:
> But isn't in a horizontal container?

I don't understand this comment.

Reply:
> I don't understand this comment.

I fixed my spelling if that was the source of the confusion.  The `.nx-page` contains two elements side-by-side, so it is a horizontal container not a vertical container, correct?

Reply:
> > I don't understand this comment.
> 
> I fixed my spelling if that was the source of the confusion. The `.nx-page` contains two elements side-by-side, so it is a horizontal container not a vertical container, correct?

Yes. You are right. Changed in dee00bb9e1484775afa1f6a0c9164206120d8358